R/coerce_to_messydate.R

```diff
@@ -44,57 +45,67 @@ as_messydate.POSIXlt <- function(x) {
 #' @export
 as_messydate.character <- function(x) {
   d <- standardise_date_separators(x)
+  d <- standardise_months(d)
   d <- standardise_date_order(d)
-  d <- standardise_date_input(d)
-  d <- standardise_unspecifieds(d)
-  d <- standardise_widths(d)
   d <- standardise_ranges(d)
+  d <- standardise_unspecifieds(d)
   d <- remove_imprecision(d)
+  d <- standardise_date_input(d)
+  d <- standardise_widths(d)
```

Comment:
Why do these come later in the sequence? Does this result in an important change for how the function operates?

Reply:
These come later since the first few standardising helper functions are about separators, component order, and unspecified components to them deal with standardising negative dates and widths (adding zeros where necessary). This does not alter how the function works in its' essence, I believe they were reshuffled to avoid redundancy. Please let me know if you would like me to change these around.